__tests__/app.test.js

```diff
@@ -29,20 +29,59 @@ describe("app", () => {
           });
         });
     });
-    test("400: handles the error", () => {
-      return request(app).get("/api/notAPath").expect(404);
-      // .then((response) => {
-      //   const {
-      //     body: { msg },
-      //   } = response;
-      //   expect(msg).toBe("Not a valid path");
-      // });
+  });
+  describe("2. GET: /api/reviews/:review_id", () => {
+    test("200: responds with a review object containing properties", () => {
+      const review_id = 2;
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then((response) => {
+          const reviews = response.body.review;
+          console.log(reviews);
+          expect(reviews).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              review_body: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              category: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
     });
   });
 });
 
-// describe("Error handling", () => {
-// 	test("1.GET /api/notAPath should respond with status 404 and display a message", () => {
-// 		return request(app).get("/api/topickssS").expect(404);
-// 	});
-// });
+describe("Error handling", () => {
+  describe("1. GET /api/categories", () => {
+    test("404: handles the error", () => {
```

Comment:
This is a very vague test description. Also, this error is not specific to the categories endpoint. Although this is included in the categories ticket, this 404 error will actually be applicable for any paths received that aren't dealt with by our endpoints.

Reply:
I'd also like to you expect a message back that will tell the client what's wrong with their request, please!